github_pr.py

```diff
@@ -37,6 +40,8 @@ def _print_prs_table(prs, **args):
         print tabulate(prs_data, headers=table_headers, tablefmt=args['tableformat'])
 
 def _print_pr(pr, **args):
+    if 'filters' in args and args['filters']:
+        pr = pr['pr']
```

Comment:
so, if `filters` are applied, we don't pass the actual pr object into this function? I think we should keep the behavior of this function consistent